components/director/internal/domain/api/resolver.go

```diff
@@ -147,3 +160,38 @@ func (r *Resolver) DeleteAPIAuth(ctx context.Context, apiID string, runtimeID st
 
 	return convertedOut, nil
 }
+
+func (r *Resolver) FetchRequest(ctx context.Context, obj *graphql.APISpec) (*graphql.FetchRequest, error) {
+	if obj == nil {
+		return nil, errors.New("Document cannot be empty")
```

Comment:
Why does this error state that the document cannot be empty? Which document?

Reply:
Right :/ I will fix it ASAP